src/Data/ProgrammesDb/EntityRepository/ServiceRepository.php

```diff
@@ -53,4 +53,20 @@ public function findAllInNetworks(): array
 
         return $qb->getQuery()->getResult(Query::HYDRATE_ARRAY);
     }
+
+    public function findAllInNetwork($networkId): array
+    {
+        $alias = 's';
```

Comment:
no need to have this be a variable if you're never going to fiddle with it, always call it `service`